src/sigmatcher/analysis.py

```diff
@@ -108,47 +62,55 @@ def filter_signature_matches(matches_per_signature: Iterable[Tuple[Signature, Li
     return whitelist_matches
 
 
+def check_match_count(matches: Set[T], analyzer_name: str) -> None:
```

Comment:
This smells like a method of `Analyzer`. Why isn't it?

Reply:
Because I'm wrong sometimes?